kafka-bin-scripts/README.adoc

```diff
@@ -94,6 +95,7 @@ To enable the Kafka scripts to access a Kafka instance, you must configure the c
 .Prerequisites
 ifndef::qs[]
 * You have the generated credentials for your service account. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots), clicking *Connection*, and creating the service account.
+* You've set the permissions for your service account to access the Kafka instance resources. You used the *Permissions* tab of the Kafka instance to set these permissions after you created the service account.
```

Comment:
I'm wondering if there's a good way to streamline this and make it clear that both of these prereqs are related to the service account that you created for the Kafka instance ...

- Are we assuming that the user has used the UI to create the service account and set permissions?
- Are we assuming that the same user who created the instance and service accounts is using this (or any other post-Getting Started quick start)?

If yes to both, maybe it becomes something like:
- You've created a service account and assigned it permissions to access the Kafka instance resources. 
   - For help creating service accounts and assigning permissions, see [link]
- You gave that service account permissions to access the Kafka instance resources

Reply:
See my previous comment. We're limited by the fact that we can't use cross-refs in quick start content that appears in the UI, and prereqs will be in the UI soon.